app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/HelpCenterPresenter.kt

```diff
@@ -138,21 +143,35 @@ internal class HelpCenterPresenter(
       if (quickLinksUiState !is HelpCenterUiState.QuickLinkUiState.QuickLinks) {
         quickLinksUiState = HelpCenterUiState.QuickLinkUiState.Loading
       }
-      quickLinksUiState = getQuickLinksUseCase.invoke().fold(
-        ifLeft = {
-          HelpCenterUiState.QuickLinkUiState.NoQuickLinks
-        },
-        ifRight = { quickActionList ->
-          val list = quickActionList.map { quickAction ->
-            HelpCenterUiState.QuickLink(quickAction)
-          }.toNonEmptyListOrNull()
-          if (list == null) {
+      combine(
+        flow = flow { emit(getQuickLinksUseCase.invoke()) },
+        flow2 = flow { emit(getHelpCenterFAQUseCase.invoke()) },
+      ) { quickLinks, faq ->
+        quickLinksUiState = quickLinks.fold(
+          ifLeft = {
             HelpCenterUiState.QuickLinkUiState.NoQuickLinks
-          } else {
-            HelpCenterUiState.QuickLinkUiState.QuickLinks(list)
-          }
-        },
-      )
+          },
+          ifRight = { quickActionList ->
+            val list = quickActionList.map { quickAction ->
+              HelpCenterUiState.QuickLink(quickAction)
+            }.toNonEmptyListOrNull()
+            if (list == null) {
+              HelpCenterUiState.QuickLinkUiState.NoQuickLinks
+            } else {
+              HelpCenterUiState.QuickLinkUiState.QuickLinks(list)
+            }
+          },
+        )
+        val topics = faq.getOrNull()?.topics ?: listOf()
+        val questions = faq.getOrNull()?.commonFAQ ?: listOf()
+        currentState = currentState.copy(
+          topics = topics,
+          questions = questions,
+          quickLinksUiState = quickLinksUiState,
+          selectedQuickAction = selectedQuickAction,
+          showNavigateToInboxButton = hasAnyActiveConversation,
+        )
+      }.collectLatest {}
```

Comment:
This can be `.collect()` since the `latest` part doesn't take any effect here with an empty body